bin/bin/applications/radio

```diff
@@ -58,8 +58,9 @@ play(){
         exit 1
     fi
 
-    cover_art_path="$cache_dir/$(printf %s "$station_name" | tr -d ' ').jpg"
     cover_art_url=$(printf %s "$radio_stations" | jq -r ".\"$station_name\".coverArtUrl")
+    cover_art_ext=$(echo "${cover_art_url##*.}")
```

Comment:
Could you change `echo` to `printf`?

Reply:
replaced in [0017d4c](https://github.com/sdushantha/dotfiles/pull/8/commits/0017d4cdf700e24238a9f913fa2e6d5a02b87a8a)